### ISSUE TITLE
use symbols rather than hex to refer to data elements

### ISSUE THREAD

**Issue by pieper** [dcmjs-org/dicom-microscopy-viewer#27]
I think this code (and all DICOM code) would be more readable and maintainable if we avoided the use of hex and use the dictionary.  Since we have control over the dictionary class this is a well defined mapping even if the standard is updated and the terminology changes (this happens only very rarely).

So this code:
```
dict = dcmjs.data.DicomMetaDictionary
dict.nameMap.FrameOfReferenceUID.tag
```
Gives the string `"(0020,0052)"` which can be passed to `dict.unpunctuateTag` to get `00200052` for use with the metadata in [code like this](https://github.com/dcmjs-org/dicom-microscopy-viewer/blob/master/src/metadata.js#L14).

We can also look at maybe adding something like `dict.rawTags.FrameOfReferenceUID` to further simplify the code.

**Comment by pieper** [comment]
And as we improve dcmjs to cover more of the standard (TIDs and CIDs) we should consider the corresponding issues being implemented in [pydicom](https://github.com/pydicom/pydicom/pull/824) and ideally work from some common json representations.

**Comment by hackermd** [comment]
@pieper I agree. The question is whether we should introduce *dcmjs* as a dependency.
Alternatively, we could simply include a JSON lookup object (as described in the [DICOM documentation](https://www.dicomstandard.org/dicomweb/dicom-json-format/)).

**Comment by pieper** [comment]
Yes, the JSON lookup object looks nice and would be lighter weight, but dcmjs is small too and may provide other useful helpers.  I'm really fine either way - just want the code to be more readable.

**Comment by hackermd** [comment]
I think we may ultimately need to (or may want to) use *dcmjs* for other purposes as well to not duplicate code across repositories (e.g., UID generation).

**Comment by hackermd** [comment]
Addressed with 1c86f0e5b73dcbb92614c975800feb3208bff195. Available since release [0.4.1](https://github.com/dcmjs-org/dicom-microscopy-viewer/releases/tag/v0.4.1)
